cli/internal/cmd/iamcreate_test.go

```diff
@@ -73,7 +73,7 @@ func TestIAMCreateAWS(t *testing.T) {
 		fs := afero.NewMemMapFs()
 		fileHandler := file.NewHandler(fs)
 		for _, f := range existingFiles {
```

Comment:
```suggestion
		for _, f := range existingConfigFiles {
```
Should be renamed to be more specific. Otherwise, it seems awkward to write config data as general-purpose dummy data.